.pre-commit-config.yaml

```
@@ -47,6 +47,7 @@ repos:
           - flake8-broken-line
           - flake8-comprehensions
           - pep8-naming
+          - flake8-docstrings
```

Please keep lists of dependencies in alphabetical order to make it easy to spot missing dependencies and difficult to add duplicates.
```suggestion
          - flake8-comprehensions
          - flake8-docstrings
          - pep8-naming
```